Bricks/file/file.h

```diff
@@ -54,12 +54,17 @@ namespace current {
 // Platform-indepenent, injection-friendly filesystem wrapper.
 struct FileSystem {
 #ifndef CURRENT_WINDOWS
-  static constexpr char PathSeparatingSlash = '/';
+  static constexpr char PathSeparator = '/';
```

Comment:
I googled more, and in fact what we're saying here is "directory separator", while "path separator" separates multiple paths in a path list.

The broader communities are the ones of established cross-platform libraries .NET (Path.DirectorySeparatorChar), Java (File.separator), and the minorities like Node (it's "sep" documented as "platform-specific path segment separator"), PHP (DIRECTORY_SEPARATOR), and Ruby (SEPARATOR).

I thought you favored grepping. Anyway, it's your own library, and it's your right to decide whether to follow the established conventions or not.